app/components/shared/inputs/ToggleInput.m.less

```diff
@@ -46,7 +26,7 @@
   background-color: @white;
 
 }
-.night-theme {
+:global(.night-theme) {
```

Comment:
I'm a CSS module noob.  Can you explain what is going on here?

Reply:
this puts `.night-theme` in the global scope, which means the identifying module hash won't be appended to it on compile because it's not a local style on the module

Reply:
Ah got it.  ty